pyv/clike.py

```diff
@@ -114,8 +114,8 @@ def visit_BinOp(self, node: ast.BinOp) -> str:
         op: str = self.visit(node.op)
         right: str = self.visit(node.right)
 
-        left_type: str = self._typename_from_annotation(node.left)
-        right_type: str = self._typename_from_annotation(node.right)
+        left_type: str = get_inferred_v_type(node.left)
```

Comment:
`self._typename_from_annotation` has additional magic to handle things like `Dict[Dict[int, str], float]` etc. What problem is this line fixing?

Reply:
Reverted, I tried that initially before tweaking with the inference.